Lib/fontTools/ttLib/tables/O_S_2f_2.py

```diff
@@ -354,6 +358,11 @@ def recalcAvgCharWidth(self, ttFont):
         self.xAvgCharWidth = avg_width
         return avg_width
 
+    def padVendorID(self, achVendorID):
```

Comment:
I don't think we need to have this staticmethod here, calling ljust inline should to the trick

Reply:
That was mostly to make it easier to test rather than instantiating the entire table through FontBuilder.